department.rb

```diff
@@ -0,0 +1,42 @@
+require './employee.rb'
+
+
+class Department
+  def initialize (department_name)
+    @total_salery = 0
```

Comment:
It would appear that this is an instance variable that you made and decided not to keep around, so it should probably be deleted.